web/lib/provider/session.js

```diff
@@ -58,7 +58,7 @@ const AuthProvider = ({ children }) => {
 
       if (authType === 'simple') {
         dispatch(initialVersion())
-      } else {
+      } else if (authType === 'oauth') {
```

Comment:
What is the relationship between this change and the server not running?

Reply:
It works for development vironment in the web UI.